packages/frontend/editor-ui/src/features/dataStore/types.d.ts

```diff
@@ -0,0 +1,21 @@
+import type { BaseResource } from '@/Interface';
```

Comment:
Why does this have the extension of `.d.ts` instead of just `.ts`?

Reply:
I guess this is the appropriate extension since it doesn't contain any runtime code. I would like us to switch to this convention for files with only type declarations